### ISSUE TITLE
I have problems when using the cli decorator

### ISSUE THREAD

**Issue by silpatflogan** [hugapi/hug#808]
I have the following script 

```
import hug

def find_feed_name_from_email(email):
    qry = """SELECT feed_name FROM users WHERE email = '{}' ORDER BY id""".format(email)
    row = engine.execute(qry).first()
    print(qry)

@hug.cli()
@hug.get(examples="email=yayakuri@hotmail.com")
@hug.local()
def get_report_by_email(email: hug.types.text, hug_timer=3):
    feed_name=find_feed_name_from_email(email)
    print(feed_name)

if __name__ == '__main__':
     get_report_by_email.interface.cli()
```

But, when I execute this...

`hug -f hug_cli_test.py -c get_report_by_email "asdf"`

 I get the following

`SELECT feed_name FROM users WHERE email = '/Users/username/Library/Caches/pypoetry/virtualenvs/env/bin/hug' ORDER BY id`

Is there something That I am not doing right?

I'm running this whit poetry in macOS

**Comment by timothycrosley** [comment]
This should be fixed in the just released Hug 2.5.5, thank you for reporting!

**Comment by silpatflogan** [comment]
Thank you, it did!
